ui/raidboss/data/02-arr/raid/t9.js

```diff
@@ -154,7 +154,7 @@
       alertText: {
         en: 'Heavensfall',
         de: 'Himmelssturz',
-        fr: 'Chutes du ciel',
+        fr: 'Chute du ciel',
```

Comment:
https://github.com/quisquous/cactbot/blob/master/CONTRIBUTING.md#trigger-text are my general guidelines.

In particular:
> Tell the player what to do rather than the mechanic name, i.e. prefer Get Out vs Iron Chariot
> If multiple strategies exist, tell the player the mechanic (Jail on YOU) instead of dictating a strategy.

Or, in this case, it's "if the directions are too confusing / complicated to put in a trigger, tell the player the mechanic".  Heavensfall means "get out of the middle but not too far out of the middle", but that's too long.  So, I think better to call it the mechanic name itself rather than describing what the mechanic does, and make this `Destruction universelle`.

Reply:
If these are your guidelines, I follow them.
It disturbed me and I was only having doubts.
Thank you for your help.